lib/pbench/agent/tool_meister_client.py

```diff
@@ -224,3 +229,76 @@ def terminate(self, group, interrupt=False):
                 )
                 ret_val = 1
         return ret_val
+
+
+def main() -> int:
+    """Main program for the Tool Meister client CLI.  The command line
+    arguments are:
+
+      group - the tool group on which the actions will be taken
+
+      directory - the directory where data gathered from the actions will be
+                  stored
+
+      action - the particular action to take, can we one of "start", "stop",
+               or "send" (see `cli_tm_allowed_actions`).
+    """
+    PROG = os.path.basename(sys.argv[0])
+    logger = logging.getLogger(PROG)
+
+    if os.environ.get("_PBENCH_TOOL_MEISTER_CLIENT_LOG_LEVEL") == "debug":
+        log_level = logging.DEBUG
+    else:
+        log_level = logging.INFO
+    logger.setLevel(log_level)
+    sh = logging.StreamHandler()
+    sh.setLevel(log_level)
+    shf = logging.Formatter(f"{PROG}: %(message)s")
+    sh.setFormatter(shf)
+    logger.addHandler(sh)
+
+    try:
+        group = sys.argv[1]
+    except IndexError:
+        logger.error("Missing group argument")
+        return 1
+    try:
+        directory = sys.argv[2]
+    except IndexError:
+        logger.error("Missing directory argument")
+        return 1
+    try:
+        action = sys.argv[3]
+    except IndexError:
+        logger.error("Missing action argument")
+        return 1
+    else:
+        if action not in cli_tm_allowed_actions:
+            logger.error(
+                "Unrecognized action, '{}', allowed actions are: {}",
+                action,
+                cli_tm_allowed_actions,
+            )
+            return 1
+        elif action == "kill":
+            # FIXME: we need to implement the gritty method of killing all the
+            # tool meisters, locally and remotely, and ensuring they are all
+            # properly shut down.
+            return 0
```

Comment:
Do we want to retain this?  Peter is in the process of implementing a `pbench-tools-kill` command which seems like it replaces this.